### PR TITLE
Update github actions and add codecov badge

### PR DESCRIPTION
**Pull request by kmacdonald-stsci**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,9 +5,9 @@ drizzle Documentation
     :target: http://www.astropy.org
     :alt: Powered by Astropy Badge
 
-.. image:: https://travis-ci.org/spacetelescope/drizzle.svg?branch=master
-    :target: https://travis-ci.org/spacetelescope/drizzle
-    :alt: Travis CI
+.. image:: https://codecov.io/github/spacetelescope/drizzle/branch/master/graphs/badge.svg
```
Comment (jdavies-st) [inline README.rst:8]: You'll also want to add the Github actions badge, in addition to the coverage one.

File: .github/workflows/drizzle_ci.yml [hunk 2]
```diff
@@ -104,3 +107,21 @@ jobs:
...
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install bandit
+    - name: Security checks with bandit
+      run: |
+        bandit romancal -r -x tests
```
Comment (jdavies-st) [inline .github/workflows/drizzle_ci.yml:127]: This is not doing anything I suspect, as there's no `romancal` in `drizzle`.

File: .github/workflows/drizzle_ci.yml [hunk 3]
```diff
@@ -8,6 +8,9 @@ on:
       - "*"
   pull_request:
     branches:
+  schedule:
+    # Run every Monday at 6am UTC
+    - cron: '0 6 * * 1'
```
Comment (jdavies-st) [inline .github/workflows/drizzle_ci.yml:13]: Nice!  A cron run is perfect.

File: README.rst (hunk 1 above)
Comment (kmacdonald-stsci) [reply README.rst:8]: I added this badge.

File: .github/workflows/drizzle_ci.yml (hunk 2 above)
Comment (kmacdonald-stsci) [reply .github/workflows/drizzle_ci.yml]: The github actions workflow has been modified to use `tox` now and this mistake has been corrected.

**Comment by nden** [comment]
LGTM. Remove `.travis.yml` and I think it's ready to be merged.

**Review by nden — APPROVED** [review]
lgtm

**Pull request merged**